src/actions/index.tsx

```diff
@@ -24,27 +25,27 @@ export function fetchBeers() {
   };
 }
 
-export function updateBeer(id, rating) {
+export function updateBeer(id: string, rating: number) {
   const request = db
     .collection('Beers')
     .doc(id)
-    .update({ rating: rating });
+    .update({ rating });
 
   return {
     type: UPDATE_BEER,
     payload: request,
   };
 }
 
-export function addNewBeer(data, file) {
+export function addNewBeer(data: FormikValues, file: File) {
   const request = db
     .collection('Beers')
     .doc()
     .set(data)
     .then(() => 'Success');
 
   if (data.img) {
-    let refA = storage.child(data.img);
+    const refA = storage.child(data.img);
```

Comment:
Why `refA`? What does it mean?

Reply:
it's a reference to the firebase data store. I had another one for deletion before, hence the A